source/unicodeScriptData.py

```diff
@@ -885,28 +887,49 @@
 	( 0Xe0020 , 0Xe007f , "Common" ), 
 ]
 
+
+unicodeScriptRangeEnd = [ k[1] for k in scriptRanges]
```

Comment:
Could you add a doc string here please? Mention that for performance reasons this should only be created once.